Policies/Kubernetes/allowed-proc-mount-types/azurepolicy.json

```diff
@@ -0,0 +1,68 @@
+{
+  "properties": {
+    "displayName": "Control allowed Proc Mount Type on pods in a Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy controls valid ProcMount types on pods in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
```

Comment:
Clicking on https://aka.ms/k8s-psp-proc-mount is taking to NotFound page.

Reply:
I have it pointing at master, so it won't work until this pr is merged

Reply:
For link  https://aka.ms/k8s-psp-proc-mount, can we point to a public kubernetes documentation link instead of pointing to GitHub repo? Say later someone moves the files, then the links will be broken.